### ISSUE TITLE
melodic

### ISSUE THREAD

**Issue by learningxiaobai** [lihuang3/ur5_notebook#30]
hi，is there someone work it out using melodic？

**Comment by learningxiaobai** [comment]
[ERROR] [1610054610.045944855]: The kinematics plugin (manipulator) failed to load. Error: Failed to load library /opt/ros/melodic/lib//libmoveit_kdl_kinematics_plugin.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library (Poco exception = libmoveit_kinematics_base.so.1.0.7: cannot open shared object file: No such file or directory)
[ERROR] [1610054610.046215613]: Kinematics solver could not be instantiated for joint group manipulator.

can  someone help me out？

**Comment by ashishagg70** [comment]
Yes, I could run with melodic. Try this: https://github.com/ashishagg70/ur5_ROS-Gazebo_melodic

**Comment by JinchuanMa** [comment]
Melodic version, my robotic arm can't pick up red objects, have you ever encountered it?

**Comment by ghost** [comment]
> Melodic version, my robotic arm can't pick up red objects, have you ever encountered it?

Hello,did you solve the problem running on Melodic?
